From: Scott Kitterman <scott@kitterman.com>
Date: Fri, 27 Mar 2020 01:27:20 -0400
Subject: Build with setuptools instead of distutils

Origin: vendor
Forwarded: not-needed
Last-Update: 2020-03-15

  * Patch setup.py to build with setuptools so there is enough metadata for
    dirtbike to build the wheel for pip
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index d0b4ecf..665fd83 100644
--- a/setup.py
+++ b/setup.py
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # setup.py generated by flit for tools that don't yet use PEP 517
 
-from distutils.core import setup
+from setuptools import setup
 
 packages = \
 ['pep517']
